pages/index.tsx

```diff
@@ -125,6 +125,18 @@ export default function Home({ fields, posts }: Props) {
               Bear League
             </Button>
           </InternalLink>
+          <InternalLink href="/conduct">
```

Comment:
Nice - I reckon probably makes sense to put this under a menu as opposed to as a call to action button on the home page - how about in the about section of the menu ( `./components/header.tsx:245`) ?